app/server.rb

```diff
@@ -0,0 +1,18 @@
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  register Sinatra::Partial
+
+  use Rack::MethodOverride
+
+  enable :sessions
+  enable :partial_underscores
+
+  set :session_secret, "secret message"
+  set :partial_template_engine, :erb
+  set :views, Proc.new { File.join(root, "../views") }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)